azblob/highlevel.go

```diff
@@ -185,7 +185,7 @@ type DownloadFromBlobOptions struct {
 
 // downloadBlobToWriterAt downloads an Azure blob to a buffer with parallel.
 func downloadBlobToWriterAt(ctx context.Context, blobURL BlobURL, offset int64, count int64,
-	writer io.WriterAt, o DownloadFromBlobOptions, initialDownloadResponse *DownloadResponse) error {
+	writer io.WriterAt, o DownloadFromBlobOptions, initialDownloadResponse *downloadResponse) error {
```

Comment:
>downloadResponse [](start = 73, length = 16)

This looks like an accidental edit from the refactoring. It should stay  DownloadResponse